astra-db-java/pom.xml

```diff
@@ -87,32 +100,21 @@
         </dependency>
 
         <!-- Working with JSON -->
-        <dependency>
-            <groupId>com.fasterxml.jackson</groupId>
-            <artifactId>jackson-bom</artifactId>
-            <version>${jackson.version}</version>
-            <scope>runtime</scope>
-            <type>pom</type>
-        </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>${jackson.version}</version>
```

Comment:
As per note above, this is no longer needed: `jackson-bom` 2.17.0 specifies 2.17.0 as default version for all ~30 Jackson components.